HTMLTestRunner3

anshumansingh229 opened the pull request.

dash0002 committed:
Updated for python3 (test_HTMLTestRunner.py still work in progress)

dash0002 committed:
no message

dash0002 committed:
Update README

dash0002 committed:
Updated content

dash0002 committed:
Added version

dash0002 committed:
Added Vagrantfile representing environment

dash0002 committed:
Refactored testing

dash0002 committed:
Updated for markdown

dash0002 committed:
Merge pull request #1 from dash0002/python3

Python3

dash0002 commented:
@anshumansingh229 Thanks for making this pull request.  My current understanding is that @tungwaiyip would like to keep his branch as-is for support of python2 and therefore my branch supports python3.  I think there's still a large pool of users for each so I wouldn't recommend having only one branch at this point.

langner commented:
Why don't we make a shared repo to coordinate efforts?

dash0002 commented:
That sounds good to me, so instead of something related to myself or @tungwaiyip we'd just have two branches?  One for python2 and one for python3?  No strong opinion either way from my side.

langner commented:
Or two separate repositories.

dash0002 commented:
From my perspective, that's what we have today.  @tungwaiyip repo for python2 and @dash0002 repo for python3.  Maybe, I'm missing the benefits of putting the repos together only to branch or separate the code bases (python 3 not being backward compatible with python 2)

langner commented:
OK

tungwaiyip commented:
Thank you for the interest in enhancing this tools. Let me know what can 
I do to facilitate the changes.

> Karol M. Langner mailto:notifications@github.com
> Monday, July 18, 2016 1:54 PM
> 
> OK
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> https://github.com/tungwaiyip/HTMLTestRunner/pull/17#issuecomment-233455467, 
> or mute the thread 
> https://github.com/notifications/unsubscribe-auth/AAXrZZ96mYYCRxpRSM2APRLLHRPpUMc0ks5qW-fugaJpZM4JOkuX.
> 
> anshumansingh229 mailto:notifications@github.com
> 
> ## Monday, July 18, 2016 3:12 AM
> 
> ```
>     You can view, comment on, or merge this pull request online at:
> ```
> 
> https://github.com/tungwaiyip/HTMLTestRunner/pull/17
> 
> ```
>     Commit Summary
> ```
> - Updated for python3 (test_HTMLTestRunner.py still work in progress)
> - no message
> - Update README
> - Updated content
> - Added version
> - Added Vagrantfile representing environment
> - Refactored testing
> - Updated for markdown
> - Merge pull request #1 from dash0002/python3
>   
>   ```
>   File Changes
>   ```
> - _A_ .gitignore
>   https://github.com/tungwaiyip/HTMLTestRunner/pull/17/files#diff-0 (1)
> - _A_ CHANGELOG.md
>   https://github.com/tungwaiyip/HTMLTestRunner/pull/17/files#diff-1 (12)
> - _M_ HTMLTestRunner.py
>   https://github.com/tungwaiyip/HTMLTestRunner/pull/17/files#diff-2 (109)
> - _A_ LICENSE.md
>   https://github.com/tungwaiyip/HTMLTestRunner/pull/17/files#diff-3 (27)
> - _D_ README
>   https://github.com/tungwaiyip/HTMLTestRunner/pull/17/files#diff-4 (5)
> - _A_ README.md
>   https://github.com/tungwaiyip/HTMLTestRunner/pull/17/files#diff-5 (68)
> - _A_ Vagrantfile
>   https://github.com/tungwaiyip/HTMLTestRunner/pull/17/files#diff-6 (76)
> - _M_ test_HTMLTestRunner.py
>   https://github.com/tungwaiyip/HTMLTestRunner/pull/17/files#diff-7 (282)
> - _A_ tests/SampleTestBasic.py
>   https://github.com/tungwaiyip/HTMLTestRunner/pull/17/files#diff-8 (96)
> - _A_ tests/SampleTestFail.py
>   https://github.com/tungwaiyip/HTMLTestRunner/pull/17/files#diff-9 (34)
> - _A_ tests/SampleTestPass.py
>   https://github.com/tungwaiyip/HTMLTestRunner/pull/17/files#diff-10
>   (21)
> - _A_ tests/**init**.py
>   https://github.com/tungwaiyip/HTMLTestRunner/pull/17/files#diff-11
>   (0)
>   
>   ```
>   Patch Links:
>   ```
> - https://github.com/tungwaiyip/HTMLTestRunner/pull/17.patch
> - https://github.com/tungwaiyip/HTMLTestRunner/pull/17.diff
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub 
> https://github.com/tungwaiyip/HTMLTestRunner/pull/17, or mute the 
> thread 
> https://github.com/notifications/unsubscribe-auth/AAXrZfqUi34M1uG_Tim-0hBWy7rgB9vNks5qW1GIgaJpZM4JOkuX.

dhruvparanjape committed:
Reduce namespace polution by only importing specific functions.

dhruvparanjape committed:
Fix sys.stdout and sys.stderr import derp

dhruvparanjape committed:
fix remaining sys import derp

dhruvparanjape committed:
Update CHANGELOG.md

dhruvparanjape committed:
Update README.md

dhruvparanjape committed:
remove unused _ variable assignment and remove unessary string slice

removed the _ that was assigned but never used and the string slice is now remove by using datetime function

dhruvparanjape committed:
Update CHANGELOG.md

dash0002 committed:
Merge pull request #3 from Dark-Passenger/master

Reduce namespace pollution by only importing specific functions.

dash0002 committed:
Update CHANGELOG.md

dhruvparanjape committed:
Add support for reporting skipped tests

Rebased @eestrada 's changes on latest HTMLTestRunner version 3.

eestrada committed:
Add support for reporting skipped tests

Rebased @eestrada 's changes on latest HTMLTestRunner version 3.

dhruvparanjape committed:
Update CHANGELOG.md

dhruvparanjape committed:
Update CHANGELOG.md

dhruvparanjape committed:
Update README.md

dhruvparanjape committed:
Update README.md

dhruvparanjape committed:
Update README.md

dhruvparanjape committed:
Update README.md

dhruvparanjape committed:
fix derp.

ns should be check for a greater than 0 explicitly rather than turthiness

dhruvparanjape committed:
fix derp.

ns should be check for a greater than 0 explicitly rather than turthiness

dhruvparanjape committed:
Update CHANGELOG.md

dhruvparanjape committed:
Update CHANGELOG.md

dhruvparanjape committed:
Add a skip test

dhruvparanjape committed:
Add a skip test

dhruvparanjape committed:
minor import cleanup and add skip testcase

add skip testcase and cleanup the imports

dhruvparanjape committed:
minor import cleanup and add skip testcase

add skip testcase and cleanup the imports

dhruvparanjape committed:
Merge remote-tracking branch 'upstream/master'

Conflicts:
	CHANGELOG.md

dhruvparanjape committed:
Merge branch 'master' of github.com:Dark-Passenger/HTMLTestRunner

dash0002 committed:
Merge pull request #4 from Dark-Passenger/master

Rebased @Dark-Passenger's, @eestrada 's changes on latest HTMLTestRunner version 3.
